src/ValueObject/Reporting/FileDiff.php

```diff
@@ -16,15 +16,17 @@
     /**
      * @var string
      * @see https://en.wikipedia.org/wiki/Diff#Unified_format
-     * @see https://regex101.com/r/AUPIX4/1
+     * @see https://regex101.com/r/sohQIG/1
```

Comment:
use increase number over generate new one, you can click "Update Regex" instead of "Save new Regex"

just on final change when this PR is ready so only change /1 to /2 so we have history on regex1.com on what the different with previous approach

Reply:
Ok no problem, I didn't want change the original one by myself (at least without your permission)